mrtt-ui/src/components/RestorationAimsForm/restorationAimsQuestionsAndAnswers.js

```diff
@@ -0,0 +1,67 @@
+export default {
+  1: {
```

Comment:
didnt want to gunk up the markup with this. Would rather keep it all in one place. Easier to refactor in the future if we have to translate or it comes from the API

Reply:
this would go in the questionMapping.js file (in data), ideally we have every section with the label and question id mapped there @mmnoo

Reply:
I was a little confused about questionMapping.js and its format to be honest. Is it just to get the `question_id` for the api?

I approached that a little differently. Im ok with `questionMapping.js` but it feels like its an implementation detail of how the projectDetailsForm was coded rather than data to be shared or a format to be enforced (given our timelines). If we had more time, I would also argue for a somewhat consistent approach for these forms, but I'm thinking we might need to let them take different approaches. (although I am interested in your approach for my own personal professional development)

It might make sense to put the `restorationAimsQuestionsAndAnswers.js` file somewhere shared and not in the component folder.